streams/cluster/src/main/java/com/hortonworks/streamline/streams/cluster/register/impl/HBaseServiceRegistrar.java

```diff
@@ -66,8 +66,7 @@ protected boolean validateServiceConfigurations(List<ServiceConfiguration> servi
     }
 
     @Override
-    protected boolean validateServiceConfiguationsViaFlattenMap(Map<String, String> configMap) {
-        // FIXME: which configuration KVs are mandatory?
+    protected boolean validateServiceConfiguationsAsFlattenedMap(Map<String, String> configMap) {
```

Comment:
nit: perhaps a better name would be: `isValidServiceConfiguationsAsFlattenedMap(...)`